src/projects/archive/index.css

```diff
@@ -2,8 +2,8 @@
     margin: 30px;
 }
 .dropdown{
-    background-color: white !important;
-    border-color: transparent !important;
-    color: black !important;
-    font-size: larger !important;
+    background-color: black;
```

Comment:
This change will likely cause merge issues for either you or for @ianrigsbee54 . When you are working on a task, please try to not push changes not relevant to your task